cmd/addimport.go

```diff
@@ -121,12 +121,147 @@ func (p *AddImportParams) SetDefaults(ctx ActionCtx) error {
 	return nil
 }
 
-func (p *AddImportParams) PreInteractive(ctx ActionCtx) error {
-	var err error
-	if err = p.AccountContextParams.Edit(ctx); err != nil {
+func (p *AddImportParams) getAvailableExports(ctx ActionCtx) ([]AccountExport, error) {
+	// these are sorted by account name
+	found, err := GetAllExports()
+	if err != nil {
+		return nil, err
+	}
+
+	ac, err := ctx.StoreCtx().Store.ReadAccountClaim(ctx.StoreCtx().Account.Name)
+	if err != nil {
+		return nil, err
+	}
+
+	var filtered []AccountExport
+	for _, f := range found {
+		// FIXME: filtering on the target account, should eliminate exports the account already has
```

Comment:
Not following this, what exactly are you filtering?

Reply:
imports that match the exports they have

Reply:
Should we really filter that? Not sure the answer is obvious TBH.